src/routes/+page.server.js

```diff
@@ -1,15 +1,20 @@
 import { getAllCollections } from '$utils/shopify';
+import { error } from '@sveltejs/kit';
 
-export async function get() {
+/** @type {import('./$types').RequestHandler} */
+export async function GET({ url }) {
   const res = await getAllCollections();
 
+  // okay this part is confusing
+  console.log(res);
+
   if (res.status === 200) {
```

Comment:
subjective, but I tend to do `if (res.ok)` rather than `if (res.status === 200)` if i trust the API not to do something wacky